### ISSUE TITLE
cannot build repository from scratch (import errors)

### ISSUE THREAD

**Issue by vrnmthr** [tatuylonen/wiktextract#35]
I downloaded the most recent version of the repo and built locally using `python3 setup.py install` but when I run `wiktwords` I get an import error.

```
vamathur@MININT-T4ROKL4:/mnt/d/src/wiktextract$ wiktwords enwiktionary-latest-pages-articles.xml.bz2 --out wikt.words --all-languages --translations
Traceback (most recent call last):
  File "/usr/local/bin/wiktwords", line 4, in <module>
    __import__('pkg_resources').run_script('wiktextract==0.3.0', 'wiktwords')
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 658, in run_script
    self.require(requires)[0].run_script(script_name, ns)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 1445, in run_script
    exec(script_code, namespace, namespace)
  File "/usr/local/lib/python3.6/dist-packages/wiktextract-0.3.0-py3.6.egg/EGG-INFO/scripts/wiktwords", line 17, in <module>
ImportError: cannot import name 'ALL_LANGUAGES'
```

**Comment by tatuylonen** [comment]
I just tried creating a brand new virtual machine (ubuntu 20.04.1 LTS) and installed from scratch, and everything seems to work.  Could you be more specific what you did to make this happen?  Could it be that you are missing wikitextprocessor (right now it needs to be installed separately; this should change once I make the release)

I installed as follows:
```
... first made sure gcc, python3-dev, python3-pip are installed
git clone https://github.com/tatuylonen/wikitextprocessor
git clone https://github.com/tatuylonen/wiktextract
(cd wikitextprocessor && pip3 install -r requirements.txt && pip3 install -e .)
(cd wiktextract && pip3 install -r wiktextract/requirements.txt && pip3 install -e .)
... then I tried extraction:
wiktwords --all --all-languages --out temp enwiktionary-20201201-pages-articles.xml.bz2
```
All went smoothly and it started to run the extraction.

**Comment by tatuylonen** [comment]
I've published version 1.99.3 on pypi.  Could you try installing now?

Installation instructions in README.md have also been updated.

I just tested by creating a fresh Ubuntu 20.04 virtual machine, using "apt update && apt install build-essential python3-dev python3-pip" and "pip3 install wiktextract", then fetched wikimedia dump using wget, and ran "wiktwords enwiktionary-20201222-pages-articles.xml.bz2" and it started running just fine.
